e2etest.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+cargo run --example e2etest
+
+if [ $? -eq 0 ];then
+exit 0
+fi
+echo "Test failed, retrying"
+
+
+cargo run --example e2etest
+
+if [ $? -eq 0 ];then
+exit 0
+fi
+echo "Test failed, retrying"
+
+cargo run --example e2etest
```

Comment:
Why is the same test run three times?

Reply:
this should no longer be necessary after #35